2023-02-11 20:22:57.758 [error] <0.629.0>@ssl_config:file_error:366 CRASH REPORT Process <0.629.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:22:57.759 [error] <0.627.0>@ssl_config:file_error:366 Supervisor {<0.627.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.629.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:22:57.760 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,1} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 1, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.459.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:21.426 [error] <0.637.0>@ssl_config:file_error:366 CRASH REPORT Process <0.637.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:23:21.427 [error] <0.635.0>@ssl_config:file_error:366 Supervisor {<0.635.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.637.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:21.427 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,2} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 2, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.460.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:41.273 [error] <0.655.0>@ssl_config:file_error:366 CRASH REPORT Process <0.655.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:23:41.273 [error] <0.653.0>@ssl_config:file_error:366 Supervisor {<0.653.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.655.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:41.273 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,3} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 3, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.461.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:51.024 [error] <0.662.0>@ssl_config:file_error:366 CRASH REPORT Process <0.662.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:23:51.024 [error] <0.660.0>@ssl_config:file_error:366 Supervisor {<0.660.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.662.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:23:51.025 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,4} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 4, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.462.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:27:53.959 [error] <0.729.0>@ssl_config:file_error:366 CRASH REPORT Process <0.729.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:27:53.959 [error] <0.727.0>@ssl_config:file_error:366 Supervisor {<0.727.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.729.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:27:53.960 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,5} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 5, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.463.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:31:11.029 [error] <0.786.0>@ssl_config:file_error:366 CRASH REPORT Process <0.786.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:31:11.030 [error] <0.784.0>@ssl_config:file_error:366 Supervisor {<0.784.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.786.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:31:11.030 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,6} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 6, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.464.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:34:56.808 [error] <0.851.0>@ssl_config:file_error:366 CRASH REPORT Process <0.851.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:34:56.808 [error] <0.849.0>@ssl_config:file_error:366 Supervisor {<0.849.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.851.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:34:56.809 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,7} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 7, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.465.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:35:30.765 [error] <0.892.0>@ssl_config:file_error:366 CRASH REPORT Process <0.892.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:35:30.766 [error] <0.890.0>@ssl_config:file_error:366 Supervisor {<0.890.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.892.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:35:30.766 [error] <0.456.0>@ssl_config:file_error:366 Supervisor {<0.456.0>,ranch_acceptors_sup} had child {acceptor,<0.456.0>,8} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 8, {sslsocket,nil,{#Port<0.12>,{config,#{key => undefined,session_tickets => disabled,handshake => full,...},...}}}, ranch_ssl, logger) at <0.466.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:44:08.533 [error] <0.574.0>@ssl_config:file_error:366 CRASH REPORT Process <0.574.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:44:08.533 [error] <0.572.0>@ssl_config:file_error:366 Supervisor {<0.572.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.574.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:44:08.533 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,1} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 1, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.454.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:53:26.619 [error] <0.614.0>@ssl_config:file_error:366 CRASH REPORT Process <0.614.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:53:26.619 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,2} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 2, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.455.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:53:26.619 [error] <0.612.0>@ssl_config:file_error:366 Supervisor {<0.612.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.614.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:53:50.591 [error] <0.618.0>@vmq_cluster_com:process_bytes:171 CRASH REPORT Process <0.618.0> with 0 neighbours crashed with reason: bad argument in vmq_cluster_com:process_bytes/3 line 171
2023-02-11 20:54:27.855 [error] <0.626.0>@ssl_config:file_error:366 CRASH REPORT Process <0.626.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 20:54:27.855 [error] <0.624.0>@ssl_config:file_error:366 Supervisor {<0.624.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.626.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 20:54:27.855 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,3} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 3, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.456.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:00:11.958 [error] <0.670.0>@ssl_config:file_error:366 CRASH REPORT Process <0.670.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 21:00:11.959 [error] <0.668.0>@ssl_config:file_error:366 Supervisor {<0.668.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.670.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:00:11.959 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,4} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 4, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.457.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:08:52.438 [error] <0.575.0>@ssl_config:file_error:366 CRASH REPORT Process <0.575.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 21:08:52.438 [error] <0.573.0>@ssl_config:file_error:366 Supervisor {<0.573.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.575.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:08:52.438 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,1} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 1, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.454.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:20:30.612 [error] <0.589.0>@ssl_config:file_error:366 CRASH REPORT Process <0.589.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 21:20:30.612 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,2} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 2, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.455.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:20:30.612 [error] <0.587.0>@ssl_config:file_error:366 Supervisor {<0.587.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.589.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:56:57.700 [error] <0.612.0>@ssl_config:file_error:366 CRASH REPORT Process <0.612.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 21:56:57.700 [error] <0.610.0>@ssl_config:file_error:366 Supervisor {<0.610.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.612.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 21:56:57.700 [error] <0.451.0>@ssl_config:file_error:366 Supervisor {<0.451.0>,ranch_acceptors_sup} had child {acceptor,<0.451.0>,3} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 3, {sslsocket,nil,{#Port<0.10>,{config,#{key => undefined,honor_ecc_order => false,secure_renegotiate => ...,...},...}}}, ranch_ssl, logger) at <0.456.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:00:20.259 [error] <0.575.0>@ssl_config:file_error:366 CRASH REPORT Process <0.575.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:00:20.259 [error] <0.573.0>@ssl_config:file_error:366 Supervisor {<0.573.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.575.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:00:20.260 [error] <0.453.0>@ssl_config:file_error:366 Supervisor {<0.453.0>,ranch_acceptors_sup} had child {acceptor,<0.453.0>,1} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 1, {sslsocket,nil,{#Port<0.11>,{config,#{key => undefined,use_ticket => undefined,handshake => full,...},...}}}, ranch_ssl, logger) at <0.456.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:07:11.026 [error] <0.582.0>@ssl_config:file_error:366 CRASH REPORT Process <0.582.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:07:11.026 [error] <0.580.0>@ssl_config:file_error:366 Supervisor {<0.580.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.582.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:07:11.026 [error] <0.453.0>@ssl_config:file_error:366 Supervisor {<0.453.0>,ranch_acceptors_sup} had child {acceptor,<0.453.0>,2} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 2, {sslsocket,nil,{#Port<0.11>,{config,#{key => undefined,use_ticket => undefined,handshake => full,...},...}}}, ranch_ssl, logger) at <0.457.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:17:28.974 [error] <0.440.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,true}] due to {already_started,<0.441.0>}
2023-02-11 22:17:28.981 [error] <0.440.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure http listener({127,0,0,1}, 8888) with Options [{max_connections,10000},{nr_of_acceptors,10},{config_mod,vmq_http_config},{config_fun,config},{proxy_protocol,false}] due to {already_started,<0.464.0>}
2023-02-11 22:22:22.381 [error] <0.443.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqttws listener({127,0,0,1}, 8888) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{proxy_protocol,false},{allowed_protocol_versions,[3,4,131]}] due to {already_started,<0.444.0>}
2023-02-11 22:22:22.385 [error] <0.443.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure http listener({127,0,0,1}, 8888) with Options [{max_connections,10000},{nr_of_acceptors,10},{config_mod,vmq_http_config},{config_fun,config},{proxy_protocol,false}] due to {already_started,<0.444.0>}
2023-02-11 22:23:05.963 [error] <0.585.0>@ssl_config:file_error:366 CRASH REPORT Process <0.585.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:23:05.963 [error] <0.582.0>@ssl_config:file_error:366 CRASH REPORT Process <0.582.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:23:05.963 [error] <0.583.0>@ssl_config:file_error:366 Supervisor {<0.583.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.585.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:23:05.963 [error] <0.479.0>@ssl_config:file_error:366 Supervisor {<0.479.0>,ranch_acceptors_sup} had child {acceptor,<0.479.0>,2} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 2, {sslsocket,nil,{#Port<0.13>,{config,#{key => undefined,handshake => full,crl_cache => {ssl_crl_cache,...},...},...}}}, ranch_ssl, logger) at <0.483.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:23:05.963 [error] <0.580.0>@ssl_config:file_error:366 Supervisor {<0.580.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.582.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:23:05.964 [error] <0.479.0>@ssl_config:file_error:366 Supervisor {<0.479.0>,ranch_acceptors_sup} had child {acceptor,<0.479.0>,1} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 1, {sslsocket,nil,{#Port<0.13>,{config,#{key => undefined,handshake => full,crl_cache => {ssl_crl_cache,...},...},...}}}, ranch_ssl, logger) at <0.482.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:23:06.018 [error] <0.590.0>@ssl_config:file_error:366 CRASH REPORT Process <0.590.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:23:06.018 [error] <0.588.0>@ssl_config:file_error:366 Supervisor {<0.588.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.590.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:23:06.019 [error] <0.479.0>@ssl_config:file_error:366 Supervisor {<0.479.0>,ranch_acceptors_sup} had child {acceptor,<0.479.0>,3} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 3, {sslsocket,nil,{#Port<0.13>,{config,#{key => undefined,handshake => full,crl_cache => {ssl_crl_cache,...},...},...}}}, ranch_ssl, logger) at <0.484.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:24:58.414 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:16.199 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:17.005 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:17.738 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:18.525 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:19.310 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:20.038 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:20.740 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:21.465 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:22.217 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:25:22.908 [error] <0.603.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-SpJxREhZXaAyRxQ3naX760h58Cc=">>},0,[<<"test">>],<<"message">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:26:14.781 [error] <0.609.0>@ssl_config:file_error:366 CRASH REPORT Process <0.609.0> with 0 neighbours crashed with reason: bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366
2023-02-11 22:26:14.781 [error] <0.607.0>@ssl_config:file_error:366 Supervisor {<0.607.0>,tls_dyn_connection_sup} had child receiver started with {ssl_gen_statem,start_link,undefined} at <0.609.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:26:14.781 [error] <0.479.0>@ssl_config:file_error:366 Supervisor {<0.479.0>,ranch_acceptors_sup} had child {acceptor,<0.479.0>,4} started with ranch_acceptor:start_link({{127,0,0,1},1883}, 4, {sslsocket,nil,{#Port<0.13>,{config,#{key => undefined,handshake => full,crl_cache => {ssl_crl_cache,...},...},...}}}, ranch_ssl, logger) at <0.485.0> exit with reason bad argument in call to erlang:binary_to_list(undefined) in ssl_config:file_error/2 line 366 in context child_terminated
2023-02-11 22:28:33.683 [error] <0.438.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,false}] due to {already_started,<0.439.0>}
2023-02-11 22:29:46.780 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:29:48.660 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:29:49.685 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:29:51.256 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:29:52.317 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:35:53.641 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:35:54.693 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:35:55.434 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:35:56.155 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:35:56.844 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:38:38.182 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:38:40.953 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:54:43.872 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:54:44.599 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:54:45.508 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:54:48.270 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 22:54:50.186 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 23:14:20.989 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"random_topic">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 23:25:05.257 [error] <0.574.0>@vmq_mqtt5_fsm:auth_on_publish:{1525,13} can't auth publish [undefined,{[],<<"anon-7NHKCw/QzObxkawMtKnWXGhvAfc=">>},0,[<<"test">>],<<"Hello World">>,false,#{}] due to no_matching_hook_found
2023-02-11 23:29:43.176 [error] <0.438.0>@vmq_ranch_config:reconfigure_listeners_for_type:{269,13} can't reconfigure mqtts listener({127,0,0,1}, 1883) with Options [{max_connections,10000},{nr_of_acceptors,10},{mountpoint,[]},{depth,1},{eccs,[sect571r1,sect571k1,secp521r1,brainpoolP512r1,sect409k1,sect409r1,brainpoolP384r1,secp384r1,sect283k1,sect283r1,brainpoolP256r1,secp256k1,secp256r1,sect239k1,sect233k1,sect233r1,secp224k1,secp224r1,sect193r1,sect193r2,secp192k1,secp192r1,sect163k1,sect163r1,sect163r2,secp160k1,secp160r1,secp160r2]},{require_certificate,false},{tls_version,'tlsv1.2'},{use_identity_as_username,false},{allowed_protocol_versions,[3,4,131]},{allow_anonymous_override,false}] due to {already_started,<0.439.0>}
